encryption_works.md

```diff
@@ -339,53 +343,54 @@ Snowden was right to be cautious and to insist that he check Gellman's new PGP f
 
-After booting to the DVD you have the option to install Tails on a USB stick, which is especially useful because it allows you to configure a [persistent volume](https://tails.boum.org/doc/first_steps/persistence/index.en.html), an encrypted section of your USB stick to store your data. Despite starting from a clean slate each time you boot up, it's important for you to be able to have access to your OTR and PGP keys, your Claws Mail (more below) and Pidgin settings, and any documents you're working with. Your persistent volume allows you to do this.
+- 1) Using the command line and the Tails .iso file to create a Tails instance on a removable storage device.
+- 2) Using an existing Tails installation and cloning the operating system onto a second storage device.
 
-### PGP and Email in Tails
+For both methods, the Tails [website](https://tails.boum.org/doc/first_steps/installation/index.en.html) has instructions.
 
-I discussed using Thunderbird with the Enigmail add-on to access your email and use PGP, however this software doesn't come with Tails. Tails comes with [Claws Mail](http://www.claws-mail.org/) which includes a PGP plugin.
+Once you get past this tricky install, Tails is currently the gold standard for anonymously using a computer, particularly when coupled with PGP-encrypted e-mail with Claws Mail, the email application bundled with Tails. For set-up instructions, check out [Claws](http://www.claws-mail.org/documentation.php?section=general) documentation.
 
-![](https://raw.github.com/micahflee/encryption-works/master/images/claws_mail.png)
+If a malicious attacker manages to modify or compromise your Tails USB stick, the security of the operating system can be compromised. For this reason, you should keep your install on you at all times and, if you suspect it's been compromised, create a new installation on a fresh device.
 
-Instead of using Enigmail's PGP key management graphical user interface to import, export, generate, view details about, and sign keys, you can click on the clipboard icon in the top right of the screen and choose Manage Keys to open [Seahorse](https://live.gnome.org/Seahorse), which provides these same features.
+Had CIA Director and retired four-star general David Petraeus and his biographer Paula Broadwell decided to use Tails, Tor, OTR, and PGP, their [extramarital affair](https://en.wikipedia.org/wiki/Petraeus_scandal) likely would have remained secret.
 
-![](https://raw.github.com/micahflee/encryption-works/master/images/tails_manage_keys.png)
+While Tails prevents many endpoint attacks, like all security tools, it is not 100% impervious to attacks. Recently, researchers have showed how attackers can implant malware in the BIOS —or the brains— of a computer, which may allow Tails to be [compromised](http://www.wired.com/2015/03/researchers-uncover-way-hack-bios-undermine-secure-operating-systems/). However, it's unclear how widespread this attack is or if it's particularly effective in the wild.
```

Comment:
IMHO, Tails is less useful for mitigating threats from malware (especially since it's apparently fairly easy to get persistent malware into the BIOS, especially the BIOS of machines from a [certain negligent manufacturer](https://trmm.net/Thunderstrike2_details)).

It is more useful because it automatically provides a significant amount of **compartmentalization**. Since you don't bring any of you old files or applications into Tails by default, you have to set everything up yourself. This encourages you to:

1. Only transfer the files and tools you need for the specific tasks you're trying to accomplish in Tails, or (even better)
2. Create compartmentalized accounts and identities (email accounts, chat accounts, GPG/OTR keys, etc.) that are totally separated from your established digital presence.